kube-insight-installation-in-GKE.md

```diff
@@ -0,0 +1,216 @@
+# kube-insight installation in Google Kubernetes Engine
+
+kube-insight repository:  https://github.com/elastisys/kube-insight-manifests
+## 1.Setup two clusters in GKE
+According to the kube-insight recommendation, Creating two clusters: one named monitored(agent) and the other named insight(server)
+
+Follow the official doc:  https://cloud.google.com/kubernetes-engine/docs/how-to/creating-a-cluster
+### 1.1. Set zone and region
+```
+li_wu@cloudshell:~ (test1012-219213)$ gcloud config set compute/zone europe-north1-a
```

Comment:
The shell prompt brings a lot of noise to each command row. I would advice to make it shorter. For instance turn

`li_wu@cloudshell:~ (test1012-219213)$ gcloud config set compute/zone europe-north1-a`

into 

`$ gcloud config set compute/zone europe-north1-a`

Reply:
... and do this on all occurrences of running a command from the prompt